include/aws/io/io.h

```diff
@@ -271,9 +254,26 @@ enum aws_io_errors {
     AWS_IO_STREAM_GET_LENGTH_FAILED,
     AWS_IO_STREAM_SEEK_UNSUPPORTED,
     AWS_IO_STREAM_GET_LENGTH_UNSUPPORTED,
+    AWS_IO_TLS_ERROR_READ_FAILURE,
 
     AWS_ERROR_PEM_MALFORMED,
 
+    AWS_IO_SOCKET_MISSING_EVENT_LOOP,
```

Comment:
Yup. Confirmed this is how it ought to be. Here's a diff of this branch on the right, and an on the left is an old io.h from commit 46974e96 (right before Dispatch Queue was merged)
<img width="717" alt="image" src="https://github.com/user-attachments/assets/5684f6c8-d187-4be0-9bf4-2f68d108935c" />